web_scraper/extract.py

```diff
@@ -75,8 +79,6 @@ def save_html_css(url: str):
     with open(os.path.join(static_folder, css_filename), 'w', encoding='utf-8') as css_file:
         css_file.write(css_content)
 
-    return html_filename, css_filename
```

Comment:
Is this line not necessary anymore? How would this data be returned?

Reply:
Yeah you don't need the line anymore, in extract.py you're only writing the files to the folder. However in load.py you actually need the names because they're used when putting the files into the S3 buckets